src/SigningPortal/SigningPortal.tsx

```diff
@@ -238,26 +258,46 @@ export const SigningPortal: React.FC = () => {
     setDryRunResult(result);
   };
 
+  // Extracts the actual Registrar call from the transaction registar, proxy, or sudo.
+  const extractRegistrarCall = (tx: UnsafeTransaction<any, string, string, any>) => {
+    let decodedCall = tx?.decodedCall;
+
+    if (isRegistrarCall(tx.decodedCall)) return tx.decodedCall;
```

Comment:
```suggestion
    if (isRegistrarCall(decodedCall)) return decodedCall;
```
Why don't we use the assigned variable here?

Reply:
https://github.com/r0gue-io/pop-wallet-signing-portal/pull/8/commits/b1e0f86033f90d8f24d8b8830ff6af2a947db387